### PR TITLE
1697 fix feature count

### PR DESCRIPTION
**Pull request by IagoLast**
Regarding to #1697.

https://team.carto.com/u/javi/builder/74bb971d-5554-459a-b67d-92834dec63a0/embed?state=%7B%22map%22%3A%7B%22ne%22%3A%5B50.31653121149523%2C6.892890930175782%5D%2C%22sw%22%3A%5B50.395825267023824%2C7.019577026367188%5D%2C%22center%22%3A%5B50.35619479483569%2C6.956233978271485%5D%2C%22zoom%22%3A13%7D%7D 

This map was rendered in . `raster mode`  because `"estimatedfeaturecount=not-available"`
this was strange because the estimatedFeatureCount was included for every layer.

After doing some research we found that the `hasEstimatedFeatureCount` function in the map object has a bug when the `layerModel.getEstimatedFeatureCount()` returns 0. (For example when the layer is hidden)

This PR changes the `layerModel.getEstimatedFeatureCount()`. (This function is now ignoring the layer's visibility) and updates the `getEstimatedFeatureCount` function in the map object to perform a single pass.

With those changes, the map is now rendered with webGL:

```
MAP RENDER MODE
{
  mode: "vector",
  reason: "webgl=yes,cartocss=supported,valid-estimated-features=9807"}
}
log.js?72c3:40 Rendered Geometries Count:  59440
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .grunt
 .rvmrc
 .tmp
+.vscode
```
Comment (xavijam) [inline .gitignore]: This doesn't belong to this PR, but ok :joy:

File: src/geo/map.js [hunk 2]
```diff
@@ -504,35 +504,37 @@ var Map = Model.extend({
   },
 
   getEstimatedFeatureCount: function () {
-    if (this.hasEstimatedFeatureCount()) {
-      return _.reduce(this.layers.getCartoDBLayers(), function (memo, layerModel) {
-        return memo + layerModel.getEstimatedFeatureCount();
-      }, 0);
+    var acum = 0;
```
Comment (xavijam) [inline src/geo/map.js]: Fine by me, but the name of the function could be changed because the purpose has changed 😬 .

**Comment by javisantana** [comment]
:es:
